src/components/LeftPanel.tsx

```diff
@@ -122,13 +124,108 @@ const LeftPanel = ({ currentTheme }: ConnectWalletProps) => {
         component="nav"
         aria-labelledby="nested-list-subheader"
       > */}
+        <div style={{ marginTop: -10 }}>
```

Comment:
This would benefit immensely from https://github.com/stacks-degens/starters-front-end/issues/45